components/dashboard/src/app/AppRoutes.tsx

```diff
@@ -96,7 +96,7 @@ export const AppRoutes = () => {
     const [isWhatsNewShown, setWhatsNewShown] = useState(user && shouldSeeWhatsNew(user));
     const newCreateWsPage = useNewCreateWorkspacePage();
     const location = useLocation();
-    const { newSignupFlow } = useFeatureFlags();
+    const newSignupFlow = !!useFeatureFlag("newSignupFlow").data;
```

Comment:
Made a comment below, but would be a bummer to push this complexity everywhere we want to check a flag. The loading state seems like it would cause the flag value to flip as well, where `data` doesn't exist initially (so `false`) and then after it comes in, might flip to true.

Reply:
We synched and decided to make the API simpler. see https://github.com/gitpod-io/gitpod/pull/17440